File: src/main/java/com/github/dylmeadows/eontimer/model/ActionSettingsConstants.java
package com.github.dylmeadows.eontimer.model;

import com.github.dylmeadows.eontimer.model.ActionMode;
import com.github.dylmeadows.eontimer.model.resources.SoundResource;
import javafx.scene.paint.Color;

public interface ActionSettingsConstants {

    ActionMode DEFAULT_MODE = ActionMode.AUDIO;

    Color DEFAULT_COLOR = Color.CYAN;

    SoundResource DEFAULT_SOUND = SoundResource.BEEP;

    int DEFAULT_INTERVAL = 500;

    int DEFAULT_COUNT = 6;
}

File: src/main/kotlin/io/github/dylmeadows/eontimer/model/settings/ActionSettingsConstants.kt
@file:JvmName("ActionSettingsConstants")

package io.github.dylmeadows.eontimer.model.settings

import io.github.dylmeadows.eontimer.model.resource.SoundResource
import javafx.scene.paint.Color

@JvmField
val DEFAULT_MODE = ActionMode.AUDIO
@JvmField
val DEFAULT_SOUND = SoundResource.BEEP
@JvmField
val DEFAULT_COLOR = Color.CYAN!!

const val DEFAULT_INTERVAL = 500
const val DEFAULT_COUNT = 6
